src/events/normalEvents/interactionCreate.ts

```diff
@@ -1,6 +1,10 @@
 import { Event } from "../../utils/Event";
 
 export const event = new Event("interactionCreate", (client, interaction) => {
+	if (interaction.isAutocomplete())
+		if (client.slashCommands.find((c) => c.name === interaction.commandName))
+			void client.processAutocompleteInteraction(interaction);

```

Comment:
This is already checked in `processAutocompleteInteraction` so no need to do a double check
```suggestion
		void client.processAutocompleteInteraction(interaction);
```

Reply:
Fr :skull: